examples/platforms/simulation/radio.c

```diff
@@ -1338,6 +1338,17 @@ otError otPlatRadioEnableCsl(otInstance         *aInstance,
     return error;
 }
 
+otError otPlatRadioResetCsl(otInstance *aInstance)
+{
+    OT_UNUSED_VARIABLE(aInstance);
+
+    otError error = OT_ERROR_NONE;
+
+    sCslPeriod = 0;
+
+    return error;
```

Comment:
```suggestion
    sCslPeriod = 0;

    return OT_ERROR_NONE;
```

Reply:
fixed for enable csl implementation as well